components/PageNavigation.vue

```diff
@@ -62,12 +83,25 @@
         default: () => []
       }
     },
+    data() {
+      return {
+        authLinks: [
+          { to: this.localePath({ name: 'account' }), text: this.$t('account.profile') },
```

Comment:
Instead of using `this.localePath` use `this.$path` (from `modules/http/templates/plugin.path.js`).